website/blog/2023-11-03-release-1.5.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Release Notes - Podman Desktop 1.5
+description: Podman Desktop 1.5 has been released!
+slug: podman-desktop-release-1.5
+authors: duffy
+tags: [podman-desktop, release, kubernetes, openshift, onboarding, compose, extensions, settings]
+hide_table_of_contents: false
+image: /blog/img/podman-desktop-release-1.5/onboarding-selkies.png
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.5 Release! 🎉
+
+With this release of Podman Desktop, we're introducing **a new onboarding feature** that we hope will earn your 🦭 seal of approval! But wait... there's so much more!
+
+- **Onboarding**: Guided setup and configuration of **Podman** and **Compose**
+- **Podman 4.7.2**: [Podman 4.7.2](https://github.com/containers/podman/releases) is now included in Windows and Mac installers
+- **Command Palette**: Gain easy access to various commands via a new keyboard-driven command palette
+- **Expanded "Summary" tab for Kubernetes pods**: Go deep with extended details on Kuberentes pods in the pod "Summary" tab
+- **Environment file support**: Chart out environment variables for new containers to access on creation
+- **Enhancements to the Settings area**: Get your bearings with improved Docker compatibility mode controls
+- **Improved user experience for state changes**: No more dead reckoning on container state with improved visual indication of status
+- **Extension API improvements**: A boatload of improvements to the extension API enabling more goodness from 🦭 Podman Desktop's extensions
+
+Podman Desktop 1.5 is now available. [Click here to download it](/downloads)!
+
+![Podman-desktop-1-5-hero](img/podman-desktop-release-1.5/onboarding-selkies.png)
+
+---
+
+## Release Details
+
+<!-- DESCRIBE MAJOR CHANGES, INCLUDE IMAGES / VIDEO IF APPLICABLE -->
+
+<!-- EXAMPLE -->
+<!-- ### Podman v4.5.1 -->
+<!-- Podman Desktop 1.1 moves up to [Podman 4.5.1](https://github.com/containers/podman/releases/tag/v4.5.1). -->
+
+<!-- WHEN DESCRIBING NAVBAR SECTIONS, INCLUDE ICON -->
+<!-- EXAMPLE -->
+<!-- We've also added options in **<icon icon="fa-solid fa-cog" size="lg" />Settings > Preferences** to automatically check for and install extension updates. -->
+
+### Onboarding
+
+We are introducing a new feature providing guided flows for the initial setup of specific 🦭 Podman Desktop extensions. Release 1.5 features two new onboarding flows: Podman, and Compose. Visit **<icon icon="fa-solid fa-cog" size="lg"/>Settings > Resources** in order to start these onboarding flows.
+
+![compose-onboarding-start](https://user-images.githubusercontent.com/799683/280276847-ca0558ab-70ad-48cc-8dd5-67e3eb465a62.png)
```

Comment:
Is there a reason we don't show the Podman onboarding. I'm afraid users may be somehow lost as this is documented between lines above

Reply:
laziness. :) I will boot up the mac and grab some. this is a good point.

Reply:
okay I got some screenshots of podman onboarding now and I reworded the text to be specific about how to access each flow / which button to press / etc.